### PR TITLE
Mailinglists/xn 28

### PR DESCRIPTION
**Pull request by adamwiteszczak**

### CODE REVIEW

File: docs/content/usage/mailinglists.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Mailinglists
+description: Using the Mailinglists API
+status: Stable
+---
+
+import {CheckIcon} from '@primer/styled-octicons';
+import {Box,Flex,Label} from '@primer/components';
+import {H3} from '@primer/gatsby-theme-doctocat/src/components/heading';
```
Comment (Smudge3806) [inline docs/content/usage/mailinglists.mdx:9]: Your not using any of these

**Review by Smudge3806 — APPROVED** [review]
👍

**Review by Smudge3806 — CHANGES_REQUESTED** [review]
Remove the unused imports in the docs and good to go

File: docs/content/usage/mailinglists.mdx (hunk 1 above)
Comment (adamwiteszczak) [reply docs/content/usage/mailinglists.mdx]: Removed in latest commit.

File: docs/content/usage/mailinglists.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+| Property      | Type      | Required                      | Example                                   |
+| :--           | :--       | :-:                           | :--                                       |
+| businessXuid  | Xuid      | <CheckIcon color="green.5"/>  | 'a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e'    |
+
+
+**Usage**
+
+```javascript
+    await Xedi.Mailinglist.list("a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e");
+```
```
Comment (Smudge3806) [inline docs/content/usage/mailinglists.mdx:53]: I perhaps wouldn't call this list as the intranet will need a list endpoint that is an actual index. getByBusiness would be more appropriate

File: docs/content/usage/mailinglists.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+Update a mailing list
+
+**Parameters**
+
+| Property      | Type      | Required                      | Example                                   |
+| :--           | :--       | :-:                           | :--                                       |
+| business_id   | Xuid      | <CheckIcon color="green.5"/>  | 'd880a624-fda6-11ea-9ff6-0100f1a50c6e'    |
+| name          | String    | <CheckIcon color="green.5"/>  | 'My update List Name'                     |
+| description   | String    | <CheckIcon color="green.5"/>  | 'My updated Description'                  |
```
Comment (Smudge3806) [inline docs/content/usage/mailinglists.mdx:82]: Don't forget ID. You need an id

File: src/Services/MailingLists.ts [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                return response.data.data
+            });
+    }
+
+    /**
+     * List all mailing lists for a business
+     * @param businessXuid
+     * @returns Promise<Instance>
+     */
+    list(businessXuid: Xuid<SupportedXuid.Business>) {
```
Comment (Smudge3806) [inline src/Services/MailingLists.ts:28]:
```suggestion
    getByBusiness(businessXuid: Xuid<SupportedXuid.Business>) {
```

**Review by Smudge3806 — CHANGES_REQUESTED** [review]
Not bad. Like the way you've done the tables in the docs, really easy to read in source.

Couple of suggestions

File: docs/content/usage/mailinglists.mdx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+| Property      | Type      | Required                      | Example                                   |
+| :--           | :--       | :-:                           | :--                                       |
+| businessXuid  | Xuid      | <CheckIcon color="green.5"/>  | 'a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e'    |
+
+
+**Usage**
+
+```javascript
+    await Xedi.Mailinglist.list("a5a4acea-fbf9-11ea-9a4e-0100f1a50c6e");
```
Comment (Smudge3806) [inline docs/content/usage/mailinglists.mdx:52]: It says getByBusiness but the example says list

**Review by Smudge3806 — CHANGES_REQUESTED** [review]
One minor change and its there

File: docs/content/usage/mailinglists.mdx (hunk 5 above)
Comment (adamwiteszczak) [reply docs/content/usage/mailinglists.mdx]: Good spot, we changed that didn't we! I've updated the docs.

**Review by Smudge3806 — APPROVED** [review]

**Pull request merged**